grpc/rust/compile/BUILD

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
This package contains the `rust_tonic_compile` rule and the `rust_binary` that powers it.

Ideally, we'd get this rule from an external repo such as `rules_rust` or `rules_proto`, but they don't have `tonic` implementations yet.

We may want to relocate this rule to another repo so it can be used by other Rust proto repos.